bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -414,6 +414,9 @@ spec:
       - name: ovn-cert
         secret:
           secretName: ovn-cert
+      - name: ovn-master-metrics-cert
```

Comment:
don't need this anymore right? now that the metrics pod is in a different daemonset.